app/src/organisms/ModuleCard/useModuleIdFromRun.ts

```diff
@@ -32,7 +32,12 @@ export function useModuleIdFromRun(
     attachedModule => attachedModule.serialNumber === module.serialNumber
   )
 
-  const moduleIdFromRun = loadModuleCommands?.[moduleIndex].result.moduleId
+  //  result is possibly undefined if you send a module command when run is terminal
+  //  to a module that is not in the protocol but plugged into the robot
+  const moduleIdFromRun =
+    loadModuleCommands?.[moduleIndex] != null
+      ? loadModuleCommands?.[moduleIndex].result.moduleId
+      : ''
```

Comment:
small thing but in our code base, we use `""` for the string we use `''` with curly brackets? 

```suggestion
      : ""
```

Reply:
`prettier` changes this back to `''`

Reply:
I see.